src/libvirtApi/domain.js

```diff
@@ -770,41 +707,40 @@ export function domainGetCapabilities({ connectionName, arch, model }) {
     return call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'GetDomainCapabilities', ['', arch, model, '', 0], { timeout, type: 'ssssu' });
 }
 
-export function domainGetStartTime({
+export async function domainGetStartTime({
     connectionName,
     vmName,
 }) {
-    return cockpit.user().then(loggedUser => {
-        const options = { err: "message" };
-        if (connectionName === "system")
-            options.superuser = "try";
-
-        /* The possible paths of logfiles path, as of libvirt 9.3.0 are defined in:
-         * https://gitlab.com/libvirt/libvirt/-/blob/v9.3.0/src/qemu/qemu_conf.c#L153
-         * There libvirt defines 3 possible directories where they can be located
-         * - /root/log/qemu/ - That is however only relevant for "qemu:///embed", and not cockpit's use case
-         * - /var/log/libvirt/qemu/ - For privileged (qemu:///system) VMs
-         * - ~/.cache/libvirt/qemu/logs - For non-privileged  (qemu:///session) VMs
-         */
-        const logFile = connectionName === "system" ? `/var/log/libvirt/qemu/${vmName}.log` : `${loggedUser.home}/.cache/libvirt/qemu/log/${vmName}.log`;
+    const loggedUser = await cockpit.user();
+    const options = { err: "message" };
+    if (connectionName === "system")
+        options.superuser = "try";
 
+    /* The possible paths of logfiles path, as of libvirt 9.3.0 are defined in:
+        * https://gitlab.com/libvirt/libvirt/-/blob/v9.3.0/src/qemu/qemu_conf.c#L153
+        * There libvirt defines 3 possible directories where they can be located
+        * - /root/log/qemu/ - That is however only relevant for "qemu:///embed", and not cockpit's use case
+        * - /var/log/libvirt/qemu/ - For privileged (qemu:///system) VMs
+        * - ~/.cache/libvirt/qemu/logs - For non-privileged  (qemu:///session) VMs
+        */
+    const logFile = connectionName === "system" ? `/var/log/libvirt/qemu/${vmName}.log` : `${loggedUser.home}/.cache/libvirt/qemu/log/${vmName}.log`;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1397-20240122-150846-61fe453f-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)